docs/pages/includes/discovery/discovery-config.yaml

```
@@ -6,6 +6,10 @@ discovery_service:
     # accounts. It prevents discovered services from colliding in Teleport when
     # managing discovered resources.
     discovery_group: "disc-group"
+    # poll_interval is the cadence at which the discovery server will run each of its
+    # discovery cycles.
+    # Default: 5m
```

From a quick look around it doesn't seem like we have a robotic "Default: <value>" in other configuration docs, we seem to prefer a more natural tone.

```suggestion
    # poll_interval is the cadence at which the discovery server will run each of its
    # discovery cycles. The default is 5m.
```